catalog/task/create-pyxis-image/0.1/README.md

```diff
@@ -1,18 +1,18 @@
 # create-pyxis-image
 
-Tekton task that executes a python script to push a container image's metadata to Pyxis.
-The task expects a workspace to be mounted containing a file with the `skopeo inspect` output of the
-container that will have its metadata uploaded to Pyxis.
+Tekton task that pushes metadata for all container images in a snapshot to Pyxis. It first extracts
+the containerImages from the snapshot, then runs `skopeo inspect` on each, before finally pushing
+metadata to Pyxis.
 
-The ID of the created `containerImage` is stored as a task result.
+The ID(s) of the created `containerImage` is stored as a task result.
```

Comment:
I undestand it may be just one, but from now on we will always treat this as a list of images. So I would recommend something like this:

The IDs of the created `containerImage` Pyxis objects are stored as a task result with each ID separated by a new line.